package.json

```diff
@@ -114,15 +114,15 @@
   },
   "devDependencies": {
     "@manageiq/manageiq-api-mock": "1.3.2",
-    "angular-mocks": "1.7.3",
+    "angular-mocks": "1.7.5",
     "axios": "0.18.0",
     "bardjs": "0.1.8",
     "chai": "3.5.0",
     "chai-as-promised": "5.3.0",
-    "codeclimate-test-reporter": "0.5.0",
+    "codeclimate-test-reporter": "0.5.1",
     "coveralls": "3.0.2",
-    "express": "4.16.3",
-    "gettext-parser": "1.3.1",
+    "express": "4.16.4",
+    "gettext-parser": "3.0.0",
```

Comment:
Cc @mzazrivec 

(just FYI, no breaking changes in https://github.com/smhg/gettext-parser/blob/master/CHANGELOG.md, and `gettext:extract`, `gettext:compile` and `gettext:validate` all keep passing without any insanity in the generated files :))